static/js/src/advantage/credentials/dashboard/components/Sidebar/Sidebar.tsx

```diff
@@ -74,21 +74,28 @@ const Sidebar = () => {
       <div className="l-navigation-bar">
         <div className="p-panel is-dark">
           <div className="p-panel__header">
-            <a className="p-panel__logo" href="/credentials" target="_blank">
+            <div
+              onClick={handleClickDashboard}
```

Comment:
@usamabinnadeem-10 Why did we switch to using a `div` instead of the anchor tag? I feel like this would be bad for accessibility.

Reply:
@usamabinnadeem-10 tagging again

Reply:
@usamabinnadeem-10 We should still strive to make it accessible. Switching to a div with an onClick event to navigate the user instead of using an `anchor` tag built for that very purpose seems redundant.

Reply:
Thanks @usamabinnadeem-10 :fire: